src/main/java/graphql/GraphQL.java

```diff
@@ -72,7 +70,7 @@ public ExecutionResult execute(String requestString, String operationName, Objec
             RecognitionException recognitionException = (RecognitionException) e.getCause();
             SourceLocation sourceLocation = new SourceLocation(recognitionException.getOffendingToken().getLine(), recognitionException.getOffendingToken().getCharPositionInLine());
             InvalidSyntaxError invalidSyntaxError = new InvalidSyntaxError(sourceLocation);
-            return new ExecutionResultImpl(Arrays.asList(invalidSyntaxError));
+            return new ExecutionResultImpl(Collections.singletonList(invalidSyntaxError));
```

Comment:
This is 100% a good change but maybe you could accumulate these on something like a "cleanup" branch and submit them without functionality changes. Or even submit them as one-off PRs. It would make evaluating PRs easier. 

Same for the comment addition below.